### ISSUE TITLE
Clear page content after save. 

### ISSUE THREAD

**Issue by radekzm** [ThemeFuse/Unyson#665]
when I want edit something on page after save plugin Unyson clear all my content. Way ?

Wersja 4.2.2 | Unyson 2.3.2 
PHP 5.5.9

**Comment by radekzm** [comment]
Even when I switch eytor, from "Visual Page Bulider" to Default. In default content is empty. WP save content only from "Default editor" ?

**Comment by radekzm** [comment]
Error or bad my action is when I save the page.

**Comment by moldcraft** [comment]
Can you give me the exact steps to reproduce this?

**Comment by moldcraft** [comment]
Thank you for giving me the access details.
But I have no idea why this happens: `fw_options` key does not exist in `$_POST`, the request looks fine. I added a debug script at the beginning of `wp-admin/post.php` and it's not there (click on the screen below)

[ ![s](https://static.md/4361d91d80c69f227de94d6513b056d8.png) ](https://static.md/4361d91d80c69f227de94d6513b056d8.png)

This looks like a server (hosting) issue.

**Comment by radekzm** [comment]
I check this and I found the solution, how to fix this problem. :)

The security module of PHP "Suhosin" removed virable from $_POST.

The default configuration causes the problem:
;suhosin.request.array_index_blacklist='"+<>;() 

Working configuration:
suhosin.request.array_index_blacklist=+<>;()

This is my server and I set it up. I check te php global virable $_POST and I confirm missing virable from HTTP REQUEST.
